AttitudeManager/FlightModes/Inc/fbwa.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef ZPSW3_AM_FBWA_HPP
+#define ZPSW3_AM_FBWA_HPP
+
+#include "flightmode.hpp"
+
+namespace AM {
+
+class FBWA : public Flightmode {
+   public:
+    FBWA() = default;
+
+    AttitudeManagerInput run(const AttitudeManagerInput& input) override;
+    void updatePid() override;
+    void updatePidGains();
+};
+
```

Comment:
Next PR - please also add a function called `updateControlLimits()` - it does exactly what you think it does.

You don't need to implement it for this PR - but the next PR we will bring it in.